### ISSUE TITLE
Integer whose value is less than (length-1)

### ISSUE THREAD

**Issue by kelloggm** [kelloggm/checker-framework#48]
This comes up in a common idiom for comparing adjacent list elements, such as checking sortedness.

For instance:

```public static boolean sorted(int[] a) {
    for (int i = 0; i < a.length - 1; i++) {
      if (a[i + 1] < a[i]) {
        return false;
      }
    }
    return true;
```

**Comment by kelloggm** [comment]
Add a new type for Less Than One Less Than Length Of (LTOLTLengthOf? LTLengthMinusOneOf?)

**Comment by kelloggm** [comment]
Fixed in 1e24949c1fb9f05ea5c74b2f9ab746863ee6fc29 using the name LTOMLengthOf
